css/css-images/object-fit-containcontainintrinsicsize-png-001i.tentative.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<!--
+     Any copyright is dedicated to the Public Domain.
+     http://creativecommons.org/publicdomain/zero/1.0/
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>CSS Test: 'object-fit: contain' and 'contain-intrinsic-size' on img element, embedding a PNG image</title>
+    <link rel="author" title="Eric Portis" href="mailto:e@ericportis.com">
+    <link rel="help" href="https://github.com/w3c/csswg-drafts/issues/10116">
+    <link rel="match" href="object-fit-containsize-png-001-ref.tentative.html">
+    <style type="text/css">
+      img {
+        border: 1px dashed gray;
+        padding: 1px;
+        image-rendering: crisp-edges;
```

Comment:
`image-rendering: crisp-edges;` is not currently supported in Chrome. It should work under the vendor alias `-webkit-optimize-contrast`, but I see several bugs about it suggesting that it's not working as expected. If this is not necessary for the tests I would remove it.

Reply:
Interesting! I modeled this after the rest of the object-fit tests, which all use it. Taking a closer look with my eyes, it's indeed not working in Chrome, but the tests still pass because it's not working in the same way for backgrounds in the reference and the img elements in the tests.

I'll change to `pixelated`.